asyncapi/asgardeo/data_types.bal

```diff
@@ -141,6 +151,26 @@ public type LoginSuccessData record {
     string serviceProvider?;
     string userName?;
     string userId?;
+    AuthStep[] authSteps?;
+};
+
+# Login Failed Data
+#
+# + organizationId - Organization Id  
+# + ref - Reference  
+# + organizationName - Organization Name  
+# + serviceProvider - Service provider  
+# + authenticatingUser - User Name  
+# + userId - User Id
+# + failedStep -  A list of Authentication Steps
+public type LoginFailedData record {
+    int organizationId?;
+    string ref?;
+    string organizationName?;
+    string serviceProvider?;
+    string authenticatingUser?;
+    string userId?;
+    AuthStep authSteps?;
```

Comment:
We have defined `failedStep` in asyncapi.yaml. But here we have added `authSteps`. Please recheck

Reply:
Resolved in [e8d6609](https://github.com/ballerina-platform/asyncapi-triggers/pull/154/commits/e8d660936bbc0a411944c843eb7e01bbc72d117c)